tests/InputFiles/ExportCoordinates/Input1D.yaml

```diff
@@ -0,0 +1,18 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: ExportCoordinates1D
+# Check: execute
+# ExpectedOutput:
+#   Coordinates0.h5
+
+DomainCreator:
+  Interval:
+    LowerBound: [0]
+    UpperBound: [1]
+    IsPeriodicIn: [false]
+    InitialRefinement: [1]
```

Comment:
(I am unaware if this is a problem but) having multiple elements means some coordinates will be repeated, right? I don't know if that will necessitate any extra cleanup steps.

Reply:
Yes, coordinates will be repeated. I expect this won't be a problem, but before feeding this to SpEC I have to reformat the data anyway, so taking care of repeated coordinates would be part of that.